_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: newsletter
+title: "Newsletter #11 - Neovim 0.5"
+category: newsletter
+permalink: /news/2021/07/5
+---
+
+# Newsletter #11: Neovim 0.5
+
+
+> "The real 0.5 was the friends we made along the way."
+
+
+The long-awaited release of Neovim 0.5.0 finally happened on July 2, 2021. Due to a variety of reasons, both technical and personal (such as a certain world-wide pandemic that everyone is heartily sick of by now), this took longer than everybody had hoped for. But it's a big one! In fact, with over 4000 commits, it is so big that it broke some of the tooling for the release. So these notes will not be able to touch on each of the many changes that were made over the course of development and only focus on the most user-visible improvements, of which the biggest are
+1. Lua as a first-class scripting and configuration language,
+2. Language server protocol (LSP),
+3. Treesitter (early access).
+
+## Lua everywhere!
+
+Neovim 0.5 went far in making Lua a first-class scripting language for Neovim, both for plugin development and for user configuration.
+
+As a reminder, Lua is a small scripting language designed for embedding and is often used in, e.g., game development (Unity). Furthermore, there is a just-in-time compiler ([LuaJIT](https://github.com/LuaJIT/LuaJIT), which Neovim is built with on platforms where it is available) that can provide impressive performance on relevant tasks. In essence, Lua was chosen over other languages because it is
```

Comment:
AFAIK, lua is not used as a scripting language in unity without a third party package. You may be thinking of Cryengine. There are of course many others.

Reply:
I may not have been thinking at all ;) I'll just remove the specific example; it doesn't add anything here.